src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const isCorrectId = tabs.find(tab => tab.id === selectedTabId)
```

Comment:
IsCorrectId is redundant variable

Reply:
but tests do not pass without it
1) should highlight the first tab if selectedTabId is wrong